sql/mysql/convert.go

```diff
@@ -60,6 +63,9 @@ func FormatType(t schema.Type) (string, error) {
 		if f == TypeFloat && t.Precision > 24 || f == TypeReal {
 			f = TypeDouble
 		}
+		if t.Unsigned {
+			f += " unsigned"
```

Comment:
don't you have another way to add words without using "space" as part of the string?